dart/lib/src/sentry_attachment/sentry_attachment.dart

```diff
@@ -0,0 +1,97 @@
+import 'dart:async';
+import 'dart:typed_data';
+
+// https://develop.sentry.dev/sdk/features/#attachments
+// https://develop.sentry.dev/sdk/envelopes/#attachment
+
+typedef ContentLoader = FutureOr<Uint8List> Function();
```

Comment:
why do we need this? we dont have on Java, and we still read it async.

Reply:
mm so if Web does not have this concept, and they need to pass eg an array of byte, there's nothing to be loaded async., its everything in memory already, so I'd say the concept of `ContentLoader` isn't needed at all? or maybe I am missing something still

Reply:
Yeah, that's right. But then you can't add file loading on demand for io platforms because attachment doesn't expose a method to do so.